src/lxc/cgroups/cgfsng.c

```diff
@@ -52,10 +52,11 @@
 
 #include "cgroup.h"
 #include "cgroup_utils.h"
-#include "commands.h"
-#include "log.h"
-#include "storage.h"
-#include "utils.h"
+#include "../commands.h"
```

Comment:
Our Makefile.am explicitly does -Isrc/lxc.  We've moved files around before, so changing the #includes to be relative instead of from our explicit include path seems like a regression.

Reply:
Ah, that was just for me locally to ease my integration with vim. :) Removed.